src/luarocks/cfg.lua

```diff
@@ -137,6 +140,7 @@ local platform_order = {
    linux = 7,
    macosx = 8,
    cygwin = 9,
+   mxe = 13,
```

Comment:
this ordering is weird... please put the numbers in order (you can change the other numbers if needed)